sh2lib/sh2lib.h

```diff
@@ -0,0 +1,260 @@
+/*
+ * SPDX-FileCopyrightText: 2027-2021 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include "esp_tls.h"
+#include <nghttp2/nghttp2.h>
+
```

Comment:
Could you add C++ guards to this header file?

Reply:
Done.